.openshiftio/Jenkinsfile.setup.groovy

```diff
@@ -0,0 +1,11 @@
+
+def setupEnvironmentPre(env) {
+  sh "oc policy -n ${env} add-role-to-user view -z default"
+  sh "if ! oc get -n ${env} configmap app-config -o yaml | grep application.properties; then oc create -n ${env} configmap app-config --from-file=greeting-service/src/main/etc/application.properties; fi"
```

Comment:
```
+ oc create -n ggastald-preview-stage configmap app-config --from-file=greeting-service/src/main/etc/application.properties
error: error reading greeting-service/src/main/etc/application.properties: no such file or directory
```

Reply:
I wonder if it would be possible for the SB team to convert this file to a .yml format? Or does it need to be a .properties file for other reasons?

Reply:
I will take a look at it tomorrow to be sure, but I seriously doubt there any specific reason of having this as `.properties` instead of `.yml`